### PR TITLE
Feat: Introduce isFluid prop to Container

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-react/src/components/Container/__tests__/Container.test.tsx [hunk 1]
```diff
@@ -7,16 +7,30 @@ import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
...
 
   stylePropsTest(Container);
 
   restPropsTest(Container, 'div');
 
   it('should render text children', () => {
-    const dom = render(<Container>Hello World</Container>);
+    render(<Container data-testid={testId}>{text}</Container>);
+
+    expect(screen.getByText(text)).toBeInTheDocument();
```
Comment (literat) [inline packages/web-react/src/components/Container/__tests__/Container.test.tsx:22]: **suggestion (if-minor):** Can you please split the test? One assert for one test. You are testing `toBeInDocument` twice and IMHO this should be in a separate test.

File: packages/web-twig/src/Resources/components/Container/Container.twig
Comment (literat) [inline packages/web-twig/src/Resources/components/Container/Container.twig:1]: **question (if-minor):** Do we want to also add this feature to the twig? This is not maintenance, IMHO.

Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Container/Container.twig:1]: Until we split visual tests, we need to update Twig as well.

File: packages/web-react/src/components/Container/__tests__/Container.test.tsx (hunk 1 above)
Comment (crishpeen) [reply packages/web-react/src/components/Container/__tests__/Container.test.tsx:22]: I copypasted this from some other component :D

File: packages/web-react/src/components/Container/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 'use client';
 
+export * from './useContainerStyleProps';
```
Comment (curdaj) [inline packages/web-react/src/components/Container/index.ts:3]: abc? :D move under Container

**Review by curdaj — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]
Web and Twig LGTM.

**Review by literat — APPROVED** [review]
LGTM 👍

**Pull request merged**